addons/mail/static/src/components/rtc_layout_menu/rtc_layout_menu.scss

```diff
@@ -2,45 +2,16 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_RtcLayoutMenu {
-    display: flex;
-    flex-direction: column;
-    padding: map-get($spacers, 1);
-}
-
-.o_RtcLayoutMenu_inputText  {
-    @include text-truncate();
-}
-
-.o_RtcLayoutMenu_separator {
-    width: 100%;
-}
-
-.o_RtcLayoutMenu_radio {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    font-size: 1.3em;
-}
-
 .o_RtcLayoutMenu_radioIcon {
     width: 5rem;
```

Comment:
The image dimensions can be set directly on the SVG code (unless these must be somehow dynamic, but I don't think it's the case)
https://css-tricks.com/scale-svg/

Reply:
Updated 👍